client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/index.tsx

```diff
@@ -334,7 +341,14 @@ export default function NotificationSettings( {
 				foundDuration = durations.find(
 					( duration ) => duration.time === settings.monitor_deferment_time
 				);
-				setSelectedDuration( foundDuration );
+
+				// We need to make sure that we are not setting a paid duration if there is no license.
+				if (
+					( foundDuration?.paid_tier && hasDowntimeMonitoringPaidLicense ) ||
+					! foundDuration?.paid_tier
+				) {
```

Comment:
Since paid licenses can access both paid and unpaid durations, we can simplify this conditional like so:

```suggestion
				if (
					hasDowntimeMonitoringPaidLicense ||
					! foundDuration?.paid_tier
				) {
```

Reply:
Good catch @mattgawarecki, I have applied the suggestion. 👍